plugins/modules/mysql_user.py

```diff
@@ -415,6 +427,8 @@ def main():
         force_context=dict(type='bool', default=False),
         session_vars=dict(type='dict'),
         column_case_sensitive=dict(type='bool', default=None),  # TODO 4.0.0 add default=True
+        password_expire=dict(type='str', choices=['now', 'never', 'default', 'interval'], no_log=True),
+        password_expire_interval=dict(type='int', no_log=True),
```

Comment:
```suggestion
        password_expire=dict(type='str', choices=['now', 'never', 'default', 'interval'], no_log=True),
        password_expire_interval=dict(type='int', no_log=True),
```
why do we `no_log` them?

Reply:
no reason why we should, will remove that one.